homeassistant/components/signal_messenger/notify.py

```diff
@@ -98,10 +99,15 @@ def __init__(
         self._signal_cli_rest_api = signal_cli_rest_api
 
     def send_message(self, message: str = "", **kwargs: Any) -> None:
-        """Send a message to a one or more recipients. Additionally a file can be attached."""
+        """Send a message to one or more recipients. Additionally a file can be attached."""
 
         _LOGGER.debug("Sending signal message")
 
+        if targets := kwargs.get(ATTR_TARGET):
```

Comment:
```suggestion
       recipients = kwargs.get(ATTR_TARGET) or self._recp_nrs
```

This can be squeezed and made more Pythonic. :)